docs/lct-basic-concepts/test-cases.md

```diff
@@ -1,11 +1,14 @@
+---
+sidebar_position: 3
+---
+
 # Test Cases {#test-cases}
 
-In LCT, Test Cases map to specific functionalities of an application that you want to test. In the case of the Sample Database Application for example, a Test Case could be adding a customer or placing an order.
+A test case is a specific scenario designed to verify whether a particular aspect of your application is working correctly. In the case of the Sample Database Application, a Test Case could be adding a customer or placing an order.
 
 :::note
 
-Except for BeforeEach- and AfterEach-Cases, Test Cases in LCT are __independent of one another__. We consider it a good practice to isolate Test Cases. This way, no error of one Test Case carries over to another one, making debugging and flakiness discovery in the test design phase easier.
-By using Playwright in the background, we make sure that each test case is run in its own isolated browser context as well.
+Except for [BeforeEach- and AfterEach-Cases](./beforeeach-aftereach-cases.md), Test Cases in LCT are __independent of one another__. This means that the execution of one Test Case does not influence the execution of another Test Case and each Test Case starts without any page loaded.
```

Comment:
Let's make two sentences out of that one:
This means that the execution of one Test Case does not influence the execution of another Test Case. Each Test Case starts without any page loaded.